### PR TITLE
Change from Travis to Github Action

### PR DESCRIPTION
**Pull request by leungkinghin**

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Build with Gradle
+        run: ./gradlew clean build
+        env:
+          CTP_PROJECT_CONFIG: ${{ secrets.CTP_PROJECT_CONFIG }}
```
Comment (ahalberkamp) [inline .github/workflows/cd.yml:31]: I think these Environment variables are wrong.
The correct once`s are:
- IT_CLIENT_ID  
- IT_CLIENT_SECRET  
- IT_PROJECT_KEY  (payment-to-order-processor-7)

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      env:
+        GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+    - name: Set up JDK 1.8
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
+    - name: Build with Maven
+      run: mvn verify
+      env:
+        CTP_PROJECT_CONFIG: ${{ secrets.CTP_PROJECT_CONFIG }}
```
Comment (ahalberkamp) [inline .github/workflows/ci.yml:26]: I think these Environment variables are wrong.
The correct once`s are:
- IT_CLIENT_ID  
- IT_CLIENT_SECRET  
- IT_PROJECT_KEY  (payment-to-order-processor-7)

**Review by ahalberkamp — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (leungkinghin) [reply .github/workflows/ci.yml]: https://github.com/commercetools/commercetools-payment-to-order-processor/pull/85/commits/75921e89c1b9141e3bbfad10bf6c21c9f204c5fd

File: .github/workflows/cd.yml (hunk 1 above)
Comment (leungkinghin) [reply .github/workflows/cd.yml]: https://github.com/commercetools/commercetools-payment-to-order-processor/pull/85/commits/75921e89c1b9141e3bbfad10bf6c21c9f204c5fd

**Review by ahalberkamp — APPROVED** [review]

**Pull request merged**